docs/submission-getting-started.md

```diff
@@ -3,4 +3,15 @@ id: submission-getting-started
 title: Getting Started
 ---
 
-Placeholder
+
+After receiving your invitation email to the ARGO Platform from the Data Coordination Center (DCC), you are ready to get started submitting data to the ARGO Platform! Submission of data can start as soon as you complete your registration.
+
+## Submitting Data to the ARGO Platform
+1. Begin by [granting access](managing-program-Access) to your submission area to your colleagues. Depending on the role you wish others to take in submission, you will be asked to select permission based roles for any users added to your program.
```

Comment:
not sure if `managing-program-Access` will have link issues due to capitalization... probably not an issue but either go lowercase or double check that it works.

Reply:
done